agimus_controller_ros/agimus_controller_ros/ros_utils.py

```diff
@@ -42,12 +43,75 @@ def array_to_ros_pose(pose_array: Pose) -> npt.NDArray[np.float64]:
     return ros_pose
 
 
+def transform_to_se3(transform: Transform) -> pin.SE3:
```

Comment:
I would name it this way 
```suggestion
def transform_msg_to_se3(transform: Transform) -> pin.SE3:
```